README.md

```diff
@@ -131,3 +104,8 @@ new [security review issue](https://github.com/cncf/tag-security/issues/new?assi
 with a
 [self-assessment](/community/assessments/guide/self-assessment.md)
 .
+
+## TAG Emeritus Leaders
+<!-- cSpell:disable -->
```

Comment:
```suggestion
<!-- cSpell:disable -->

## TAG Emeritus Leaders
```

Reply:
I don't know why this would work, but changing it to match the previous implementation